mobile/ios/ios-device.ts

```diff
@@ -303,6 +306,10 @@ export class IOSDevice implements Mobile.IIOSDevice {
 		}).future<void>()();
 	}
 
+	private getInterfaceType(): any {
+		return this.$mobileDevice.deviceGetInterfaceType(this.devicePointer);
```

Comment:
deviceGetInterfaceType returns a number, should getInterfaceType return number as well?